build/ci/run_unittests.sh

```diff
@@ -16,5 +16,5 @@
 # limitations under the License.
 #
 
-docker build -f build/ci/Dockerfile.unittest -t operator-unittests .
-docker run --rm -t operator-unittests
+make build-unit-tests-image
+make run-unit-tests
```

Comment:
is there any reason we want to require `make` installation in our CI?

Reply:
isn't it a basic command in Linux environments?

Reply:
is there any reason we want to require a Linux environment in our CI?

Reply:
we not using windows as our CI

Reply:
IMO using make commands makes the run more readable